README.md

```diff
@@ -10,4 +10,10 @@ Future plans including adding the ability to:
 * Analyze and diagnose common issues
 * Automate common cloud management and maintenance tasks
 
-Documentation is maintained [HERE](http://goldstone-server.readthedocs.org/en/latest/)
+Goldstone Server is licensed under the Apached 2.0 license (see LICENSE file)
+
+Goldstone Server Enterprise components are licensed under the Solinea Software License Agreement v1 (see GSE_LICENSE.pdf)
```

Comment:
Quick question 

I see /opt/goldstone will it make sense to use the /var like most of the openstack project so if the company does have already an image they use they do not have to make changes. I remember one of the bank partitioning was re-design for this and they did make the var bigger. 

Just my 2 cents. 

Inviato da iPhone

> Il giorno 21 mar 2016, alle ore 12:30, John Stanford <notifications@github.com> ha scritto:
> 
> In README.md:
> 
> > @@ -10,4 +10,10 @@ Future plans including adding the ability to:
> >  * Analyze and diagnose common issues
> >  * Automate common cloud management and maintenance tasks
> >  
> > -Documentation is maintained [HERE](http://goldstone-server.readthedocs.org/en/latest/)
> > +Goldstone Server is licensed under the Apached 2.0 license (see LICENSE file)
> > +
> > +Goldstone Server Enterprise components are licensed under the Solinea Software License Agreement v1 (see GSE_LICENSE.pdf)
> If we put it in the submodule, it will get burried in the container's venv source tree. Not a bad idea to have it in there, but this README gets mapped into the /opt/goldstone dir. Thoughts?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>